src/linear_algebra/basic.lean

```diff
@@ -19,7 +19,7 @@ modules over a ring, submodules, and linear maps. If `p` and `q` are submodules
 means that `p ⊆ q`.
 
 Many of the relevant definitions, including `module`, `submodule`, and `linear_map`, are found in
```

Comment:
This should be updated, right?

Reply:
Sorry, I'm not sure what you would like to see here. Split out the list of definitions per-file instead of referring to the whole folder?